src/main/java/edu/msudenver/bluejext/qualitytools/QualityToolsExtension.java

```diff
@@ -91,3 +99,42 @@ public URL getURL() {
         }
     }
 }
+
+/**
+ * Create a Preference Panel located in the BlueJ application
+ * BlueJ -> Preferences -> Extensions Tab
+ */
+class Preferences implements PreferenceGenerator
+{
+    private JPanel myPanel;
+    private JTextField color;
+    private BlueJ bluej;
+    public static final String PROFILE_LABEL="Quality-Analysis-Tools";
+
+    // Construct the panel, and initialise it from any stored values
+    public Preferences(BlueJ bluej)
+    {
+        this.bluej = bluej;
+        myPanel = new JPanel();
+        myPanel.add (new JLabel ("Quality Analysis Tools"));
+        color = new JTextField (40);
+        myPanel.add (color);
```

Comment:
What is "color"?

Reply:
We will remove "color" object as it was used to store the JTextField.  There is no user input required for this user story.  Will remove lines of code associated with "color".  

For reference and to answer your question see the information below.  

`color = new JTextField (40);`
API recommends Note: 
"We encourage you to specify the number of columns for each text field. If you do not specify the number of columns or a preferred size, then the field's preferred size changes whenever the text changes, which can result in unwanted layout updates."

https://docs.oracle.com/javase/tutorial/uiswing/components/textfield.html

Reply:
Just a note that it is less about "user input" and more about the choice of "color" in particular as having no obvious relevance and likely being an unintended artifact of a copy-paste of work done by others.